script/fetch_content.py

```diff
@@ -0,0 +1,18 @@
+#Script which will take as an input url to the page and text.
+import argparse
+import requests
+
+
+parser = argparse.ArgumentParser()
```

Comment:
It would be nice to start using functions.
Please create function arguments() which will return arguments passed in the command line (url and text)

Reply:
oki :)